### PR TITLE
CoinRanking Auth

### PR DESCRIPTION
**Pull request by tannershimanek**
Updated the authentication for CoinRanking from "No" to "apiKey.

- [x] My submission is formatted according to the guidelines in the [contributing guide](CONTRIBUTING.md)
- [x] My addition is ordered alphabetically
- [x] My submission has a useful description
- [x] The description does not end with punctuation
- [x] Each table column is padded with one space on either side
- [x] I have searched the repository for any relevant issues or pull requests
- [x] Any category I am creating has the minimum requirement of 3 items
- [x] All changes have been [squashed][squash-link] into a single commit

[squash-link]: <https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit>

### CODE REVIEW

**Comment by tannershimanek** [comment]
Just a small change, but increasing accuracy

File: README.md [hunk 1]
```diff
@@ -197,7 +197,7 @@ API | Description | Auth | HTTPS | CORS |
 | [Coinlore](https://www.coinlore.com/cryptocurrency-data-api) | Cryptocurrencies prices, volume and more | No | Yes | Unknown |
 | [CoinMarketCap](https://coinmarketcap.com/api/) | Cryptocurrencies Prices | `apiKey` | Yes | Unknown |
 | [Coinpaprika](https://api.coinpaprika.com) | Cryptocurrencies prices, volume and more | No | Yes | Yes |
-| [CoinRanking](https://docs.coinranking.com/) | Live Cryptocurrency data | No | Yes | Unknown |
+| [CoinRanking](https://docs.coinranking.com/) | Live Cryptocurrency data | `apiKey` | Yes | Unknown |
```
Comment (marekdano) [inline README.md:200]: When you resolve the conflict please make sure the current url of `https://developers.coinranking.com/api/documentation/` is listed, please.

**Review by marekdano — CHANGES_REQUESTED** [review]
Thanks for the contribution @tannershimanek!
Can you please resolve the conflict?

**Comment by pawelbr** [comment]
@tannershimanek 
Don't forget to squash all the commits into the single one after committing the changes suggested by @marekdano

**Comment by matheusfelipeog** [comment]
This PR is inactive, so the API was added in another PR/commit.

More information at: #2002

**Pull request closed without merging**